### ISSUE TITLE
implement a role concept

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#14]
## User story
1. As a {HR chief}
2. I want / need {the DIdentity software to offer different roles for different employees. One Admin-Role plus an employee role for the HR-employees. }
3. So that {the access to the software is limited and the rights to access and change the Data are only in the hands of the right people}

## Acceptance criteria
* Admin-Role: has all rights
* HR-Employee: can not do mayor changes to the DIdentity software, like issuing updates of the software.
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
